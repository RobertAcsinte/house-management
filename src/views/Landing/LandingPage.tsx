import NoHouse from '../../assets/nohouse.png';
import Today from '../../assets/today.png';
import Notes from '../../assets/notes.png';
import Appointments from '../../assets/appointments.png';
import Myhouse from '../../assets/myhouse.png';
import FeatureCard from './components/featureCard/FeatureCard'

function LandingPage() {
  return (
    <>
      <FeatureCard 
        title='Effortless communication and coordination among housemates.'
        description='Experience hassle-free student living everywhere in the world with House Sharing! Our platform brings together housemates, making it a breeze to share notes and coordinate schedules for communal spaces. Say goodbye to misunderstandings and hello to seamless housemate harmony!'
      />
      <FeatureCard 
        title='Join a house'
        description='You can create a new house, join an existing house by its id or get invited to one by an already existing user!'
        image={NoHouse}
      />
      <FeatureCard 
        title="See what's going on today"
        description='On the today page, you will always see the pinned notes of the house and the appointments that were made for the current day for the house facilities'
        image={Today}
      />
      <FeatureCard 
        title='Create notes for everyone'
        description='Do you have a package to receive and you want to let others know? Or is there an important announce to be made? Just create a note and it will be available for everyone. The pinned ones will always be on top and on Today page'
        image={Notes}
      />
      <FeatureCard
        title='Make a reservation'
        description='Do you have classes in the morning and you do not know when to wake up to shower so you do not have to wait for others and be late? Or do you want to be sure that when you arrive home you will have time to cook? Book a timeslot. This way everyone can sync in using the shared facilities of the house!'
        image={Appointments}
      />
      <FeatureCard
        title='Manage your house'
        description='You can always manage the house that you are part of. You can change the name of it, add members and see the pending invitations!'
        image={Myhouse}
      />
    </>
  )
}

export default LandingPage