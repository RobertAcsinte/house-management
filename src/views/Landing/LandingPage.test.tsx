
import { expect, it } from 'vitest'
import { render } from '@testing-library/react'
import LandingPage from './LandingPage'

it('Snapshot', () => {
  const result = render(<LandingPage />)
  expect(result).toMatchSnapshot()
})