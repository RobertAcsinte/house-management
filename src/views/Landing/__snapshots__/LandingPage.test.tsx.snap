// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Effortless communication and coordination among housemates.
          </h1>
          <h2>
            Experience hassle-free student living everywhere in the world with House Sharing! Our platform brings together housemates, making it a breeze to share notes and coordinate schedules for communal spaces. Say goodbye to misunderstandings and hello to seamless housemate harmony!
          </h2>
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Join a house
          </h1>
          <h2>
            You can create a new house, join an existing house by its id or get invited to one by an already existing user!
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/nohouse.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            See what's going on today
          </h1>
          <h2>
            On the today page, you will always see the pinned notes of the house and the appointments that were made for the current day for the house facilities
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/today.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Create notes for everyone
          </h1>
          <h2>
            Do you have a package to receive and you want to let others know? Or is there an important announce to be made? Just create a note and it will be available for everyone. The pinned ones will always be on top and on Today page
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/notes.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Make a reservation
          </h1>
          <h2>
            Do you have classes in the morning and you do not know when to wake up to shower so you do not have to wait for others and be late? Or do you want to be sure that when you arrive home you will have time to cook? Book a timeslot. This way everyone can sync in using the shared facilities of the house!
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/appointments.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Manage your house
          </h1>
          <h2>
            You can always manage the house that you are part of. You can change the name of it, add members and see the pending invitations!
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/myhouse.png"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Effortless communication and coordination among housemates.
        </h1>
        <h2>
          Experience hassle-free student living everywhere in the world with House Sharing! Our platform brings together housemates, making it a breeze to share notes and coordinate schedules for communal spaces. Say goodbye to misunderstandings and hello to seamless housemate harmony!
        </h2>
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Join a house
        </h1>
        <h2>
          You can create a new house, join an existing house by its id or get invited to one by an already existing user!
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/nohouse.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          See what's going on today
        </h1>
        <h2>
          On the today page, you will always see the pinned notes of the house and the appointments that were made for the current day for the house facilities
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/today.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Create notes for everyone
        </h1>
        <h2>
          Do you have a package to receive and you want to let others know? Or is there an important announce to be made? Just create a note and it will be available for everyone. The pinned ones will always be on top and on Today page
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/notes.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Make a reservation
        </h1>
        <h2>
          Do you have classes in the morning and you do not know when to wake up to shower so you do not have to wait for others and be late? Or do you want to be sure that when you arrive home you will have time to cook? Book a timeslot. This way everyone can sync in using the shared facilities of the house!
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/appointments.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Manage your house
        </h1>
        <h2>
          You can always manage the house that you are part of. You can change the name of it, add members and see the pending invitations!
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/myhouse.png"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`toUpperCase 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Effortless communication and coordination among housemates.
          </h1>
          <h2>
            Experience hassle-free student living everywhere in the world with House Sharing! Our platform brings together housemates, making it a breeze to share notes and coordinate schedules for communal spaces. Say goodbye to misunderstandings and hello to seamless housemate harmony!
          </h2>
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Join a house
          </h1>
          <h2>
            You can create a new house, join an existing house by its id or get invited to one by an already existing user!
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/nohouse.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            See what's going on today
          </h1>
          <h2>
            On the today page, you will always see the pinned notes of the house and the appointments that were made for the current day for the house facilities
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/today.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Create notes for everyone
          </h1>
          <h2>
            Do you have a package to receive and you want to let others know? Or is there an important announce to be made? Just create a note and it will be available for everyone. The pinned ones will always be on top and on Today page
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/notes.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Make a reservation
          </h1>
          <h2>
            Do you have classes in the morning and you do not know when to wake up to shower so you do not have to wait for others and be late? Or do you want to be sure that when you arrive home you will have time to cook? Book a timeslot. This way everyone can sync in using the shared facilities of the house!
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/appointments.png"
          />
        </div>
      </div>
      <div
        class="_container_e55ede"
      >
        <div
          class="_wrapper_e55ede"
        >
          <h1>
            Manage your house
          </h1>
          <h2>
            You can always manage the house that you are part of. You can change the name of it, add members and see the pending invitations!
          </h2>
          <img
            alt="Description of the feature"
            class="_image_e55ede"
            src="/src/assets/myhouse.png"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Effortless communication and coordination among housemates.
        </h1>
        <h2>
          Experience hassle-free student living everywhere in the world with House Sharing! Our platform brings together housemates, making it a breeze to share notes and coordinate schedules for communal spaces. Say goodbye to misunderstandings and hello to seamless housemate harmony!
        </h2>
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Join a house
        </h1>
        <h2>
          You can create a new house, join an existing house by its id or get invited to one by an already existing user!
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/nohouse.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          See what's going on today
        </h1>
        <h2>
          On the today page, you will always see the pinned notes of the house and the appointments that were made for the current day for the house facilities
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/today.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Create notes for everyone
        </h1>
        <h2>
          Do you have a package to receive and you want to let others know? Or is there an important announce to be made? Just create a note and it will be available for everyone. The pinned ones will always be on top and on Today page
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/notes.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Make a reservation
        </h1>
        <h2>
          Do you have classes in the morning and you do not know when to wake up to shower so you do not have to wait for others and be late? Or do you want to be sure that when you arrive home you will have time to cook? Book a timeslot. This way everyone can sync in using the shared facilities of the house!
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/appointments.png"
        />
      </div>
    </div>
    <div
      class="_container_e55ede"
    >
      <div
        class="_wrapper_e55ede"
      >
        <h1>
          Manage your house
        </h1>
        <h2>
          You can always manage the house that you are part of. You can change the name of it, add members and see the pending invitations!
        </h2>
        <img
          alt="Description of the feature"
          class="_image_e55ede"
          src="/src/assets/myhouse.png"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
